.github/workflows/main.yml

```diff
@@ -91,13 +91,19 @@ jobs:
             //software/simulated_tests/...   \
             //software/ai/hl/...
 
-      - name: Upload simulated test artifacts
-        uses: actions/upload-artifact@v2
+      - name: Upload simulated test proto logs
+        uses: actions/upload-artifact@v3
         with:
-          name: simulated-test-outputs
-          # we only care about the replay logs from the STP tests
+          name: blue-ai-proto-logs
           path: |
-            src/bazel-out/k8-fastbuild/testlogs/software/ai/hl/stp
+            /tmp/tbots/blue/**/protolog_.*
```

Comment:
```suggestion
            /tmp/tbots/blue/
```

Reply:
thats the first thing I tried, that doesn't work either

ill play around with it a bit more

Reply:
which commit?

Reply:
https://github.com/UBC-Thunderbots/Software/pull/2601/commits/9f8d1e5d08475215d2d70d00c47d76854cf39719